chart/templates/grafana-db-sec.yaml

```diff
@@ -10,15 +9,19 @@ metadata:
     app: {{ template "tobs.fullname" . }}
     chart: {{ template "tobs.chart" . }}
     release: {{ .Release.Name }}
+  annotations:
+    "helm.sh/resource-policy": keep
 type: Opaque
+{{- if (($grafana.timescale).database).enabled }}
```

Comment:
Note: parenthesis here allow to have each level be nullable and thus prevent issues with `nil pointer evaluating interface` when higher level object doesn't exist.